platform-server/src/main/resources/application-development.yml

```diff
@@ -1,5 +1,7 @@
 coduno:
   url: http://localhost:6060
+  mail:
+    from: noreply@cod.uno
```

Comment:
We could be a little more hipster here and use a `pleasereply@` or `hi@` ... There's really no reason why people can/should not respond to that e-mail. It's very easy for them to contact us that way in case they are confused about the e-mail.